chunky/src/java/se/llbit/chunky/resources/BinaryBitmapImage.java

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2020-2021 Chunky contributors
+ *
+ * This file is part of Chunky.
+ *
+ * Chunky is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Chunky is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with Chunky.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package se.llbit.chunky.resources;
+
+/**
+ * A image type using 1 bit per pixel
+ */
+public class BinaryBitmapImage {
+  public final int width;
+  public final int height;
+  private final byte[] data;
```

Comment:
I wonder if it would be faster to use `long[]` here. Java's `BitSet` seems to use an array of long.

Reply:
My intuition is that `long` is better when you have some functions that operate on multiple bits, for example `and`, `or` and `xor` of `BitSet` or probably 8 time as fast if they use `long` compared to `byte`, but when accessing one bit at a time, I don't think there is any difference.
On the other hand, using `byte` could lead to less memory allocated (but not in this case as the number of bytes allocated is multiple of 8).
That being said, if you want some measurements, I can do that